.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,15 +4,15 @@
 
 #### General
 
-- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
+- [ ] Update Changelog following the conventions laid out on [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
```

Comment:
I'd remove the extra space after Guidelines. Not sure if it impacts the formatting but better to keep the markdown tight.